streaming_test.go

```diff
@@ -6,10 +6,100 @@ import (
 	"io/ioutil"
 	"sync"
 	"testing"
+	"time"
 
 	"github.com/valyala/fasthttp/fasthttputil"
 )
 
+func TestStreamingPipeline(t *testing.T) {
```

Comment:
Streaming wasn't compatible with pipelining as the stream would already read data from the next request which would then fail the next request.